content/en/docs/tutorials/services/source-ip.md

```diff
@@ -27,10 +26,11 @@ This document makes use of the following terms:
 * [Kube-proxy](/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies): a network daemon that orchestrates Service VIP management on every node
 
 
-## Prerequisites
+### Prerequisites
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
```

Comment:
👋  @sftim 
This change looks interesting. A couple of thoughts:
Adding the minimum required `server` version in the page's front matter sets the value in the `version-check` shortcode.

There are a number of places on this page where the minimum required version is repeated. Would it make sense to use this variable?
Do these sentences mean the same thing (The first sentence comes from text in `en.toml`)?
```
Your Kubernetes server must be at or later than version v1.5. To check the version, enter kubectl version.
vs.
You must have a working Kubernetes 1.5 cluster to run the examples in this document.
```

Reply:
Good point - there's more to revise.